docs/source/conf.py

```diff
@@ -493,41 +492,8 @@ def autodoc_process_docstring(app, what, name, obj, options, lines):  # noqa: PL
     remove_arrows_in_examples(lines)
 
 
-def env_override(default_appid):
-    build_version = os.getenv("READTHEDOCS_VERSION")
-
-    if build_version == "latest":
-        return os.environ["HEAP_APPID_QA"]
-    if build_version == "stable":
-        return os.environ["HEAP_APPID_PROD"]
-
-    return default_appid  # default to Development for local builds
-
-
-def _add_jinja_filters(app):
```

Comment:
Thinking that maybe these functions should stay here, instead of being in the theme itself